test/cluster_resource_test.go

```diff
@@ -176,13 +176,13 @@ clusters:
 - cluster:
     certificate-authority-data: WTJFdFkyVnlkQW89
     server: https://1.1.1.1
-  name: helloworld-cluster
+  name: target-cluster
 contexts:
 - context:
-    cluster: helloworld-cluster
+    cluster: target-cluster
     user: test-user
-  name: helloworld-cluster
-current-context: helloworld-cluster
+  name: target-cluster
+current-context: target-cluster
```

Comment:
is there a test that also uses the deprecated one, to make sure it works properly?

Reply:
Added one!